articles/quickstart/native/android/04-user-profile.md

```diff
@@ -49,11 +51,15 @@ AuthenticationAPIClient authClient = new AuthenticationAPIClient(auth0);
 ```
 
 ::: note
-It's suggested that you add both the Auth0 `domain` and `clientId` to the `strings.xml` file rather than hardcode them.
+Do not hardcode the Auth0 `domain` and `clientId` values. We recommend you add them to the `strings.xml` file.
 :::
 
-Next, use the `access_token` to obtain the user id with the `AuthenticationAPIClient`. Although the call returns a `UserProfile` instance, this is a basic OIDC conformant profile and the only guaranteed claim is the `sub` which contains the user's id, but depending on the requested scope the claims returned may vary. With the `sub` value call the [Management API](https://auth0.com/docs/api/management/v2#!/Users) and get a complete user profile back.
+To get the user's information:
 
+1. Use the user's access token to create an instance of the `AuhenticationAPIClient` client.
```

Comment:
@aaguiarz, could you take a look at the revised instructions to see if they're ok?

Reply:
The client itself doesn't require an `access_token` to be instantiated. What we meant was that we're going to use an instance of the API client to call one of its methods (userInfo) that receives an `access_token` and returns the OIDC-conformant profile of the user associated to that token.